tests/utils/CMakeLists.txt

```diff
@@ -14,9 +14,6 @@ add_executable(float_bin_to_int8 float_bin_to_int8.cpp)
 
 add_executable(ivecs_to_bin ivecs_to_bin.cpp)
 
-add_executable(count_bfs_levels count_bfs_levels.cpp)
```

Comment:
https://github.com/microsoft/DiskANN/blob/harshasi/pybind_diskann/tests/utils/CMakeLists.txt#L15 because you had it removed in your branch.  it seems like it was removed from the source code entirely in your branch - do you remember having a reason for it, or was it an accident?